src/interfaces/index.d.ts

```diff
@@ -1,4 +1,5 @@
 export * from "./buttons";
+export * from './card';
```

Comment:
@kamilmateusiak do we want to export `Divider` interface here?

Reply:
woaaah, nice catch :) @dprzybylek can you add `export * from './divider';` here?

Reply:
yup, on the way

Reply:
done at 9f8670cd9d06aeb8e3f8825b50603d60fca22f62